templates/build-image.Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Dockerfile to bootstrap build and test in openshift-ci
+
+FROM openshift/origin-release:golang-__GOLANG_VERSION__
+
+# Add kubernetes repository
+ADD openshift/ci-operator/build-image/kubernetes.repo /etc/yum.repos.d/
+
+RUN yum install -y kubectl ansible httpd-tools
+
+RUN GO111MODULE=on go get github.com/mikefarah/yq/v3
```

Comment:
Can we install `yq` the same way for the action above? We had issues with different formats in the past sadly.

Reply:
I don't get you here. What are you proposing. This file is just a template taken from source that was just like this. https://github.com/openshift-knative/serverless-operator/blob/fb56d4c/openshift/ci-operator/build-image/Dockerfile#L10

Reply:
I think Markus is referring to using `go get` vs `wget` as you do in the validate.yaml.

Reply:
A yes. But, I think that's better to stick with wget as that will result in predictive version being used. `go get` can change version, and break something unpredictably.

Reply:
I think it's possible to use `go get` with a version specified but I couldn't download 3.4.0 this way.

Reply:
Also, using wget/curl is faster and simpler. It doesn't require to have Golang setup & configured.